packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -68,22 +70,34 @@ export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
     gridTiles.push(localVideoComponent);
   }
 
+  const horizontalGallery = useMemo(() => {
+    if (horizontalGalleryTiles.length === 0) {
+      return null;
+    }
+    /* @conditional-compile-remove(pinned-participants) */
+    if (isNarrow) {
+      return <ScrollableHorizontalGallery horizontalGalleryElements={horizontalGalleryTiles} />;
+    }
+    return (
+      <VideoGalleryResponsiveHorizontalGallery
+        isNarrow={isNarrow}
+        shouldFloatLocalVideo={true}
```

Comment:
The local video looks to be occluding the other tiles (sits on top of them) but I cannot drag it off the tile to see behind it in storybook:
![image](https://user-images.githubusercontent.com/2684369/211356974-d14a22b3-be16-4bdb-9931-1775e2071b11.png)

Reply:
From the figma this looks correct, but I assume only because it can be dragged away:
![image](https://user-images.githubusercontent.com/2684369/211379115-445e0908-8212-4819-a71b-621cc7c67913.png)

Reply:
should we then file this as a bug to finally make the local tile draggable in mobile again? I think it should be doable now that we have that modal

Reply:
@JamesBurnside You can now drag the scrollable horizontal gallery with your mouse. I had to use this nifty package https://www.npmjs.com/package/react-use-draggable-scroll

Reply:
I am still unable to drag it on storybook using mouse - are you able to?

https://user-images.githubusercontent.com/2684369/211616408-99b13386-b558-400e-8b7f-fae5b8f7e299.mp4

Is there a reason we needed to use [npmjs.com/package/react-use-draggable-scroll](https://www.npmjs.com/package/react-use-draggable-scroll) and not use the fluent modal -- the reason to try use the fluent modal is to avoid multiple implementations for the same behavior (each will have their own nuances/bugs so better if we can keep it consistent)